docs/proposals/scheduling/20230707-eviction-arbitrate-mechanism-in-descheduler.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Eviction Arbitration Mechanism in Descheduler
+authors:
+  - "@baowj-678"
+reviewers:
+  - @hormes
```

Comment:
The format is incorrect. Missing double quotes.

Reply:
ok